lib/eco/treemanager.py

```diff
@@ -134,6 +137,8 @@ def jump_right(self):
         if self.inside() or isinstance(node, EOS):
             self.pos = len(self.node.symbol.name)
             return
+        if node.symbol.name == "\r":
+            self.line += 1
```

Comment:
I suspect this means a rebase is in order before merge (indeed the branch is marked as having conflicts with master).